README.md

```diff
@@ -62,6 +66,46 @@ $ npm run start:dev
 
 ```
 
-### Automated monitoring
+## Instructions for anchors
+
+### Interactive flow instrumentation
+
+The interactive flow of SEP24 is custom for each anchor which makes it difficult
+to automate. In order to help the automated tests complete your interactive
+flow, any form fields need to be annotated with expected values.
+
+There are two things that need to be done: provide valid values for each field,
+and identify which button should be pressed to continue.
+
+#### Providing field values
+
+Any form field that is required should have a `test-value` attribute which is
+set to a valid value for that field. For example an email field would look like
+`<input type='text' id='email_address' test-value='dummyaddress2342@gmail.com' />`.
+The number can be randomized at render time to avoid reuse.
+
+An example of doing this using Polaris's form stack can be seen
+[here](https://github.com/stellar/django-polaris/blob/fd5900d68fec6b0e31ce720262e8d787fcbf8aac/example/server/forms.py#L10,L15)
+but any framework should be able to add these attributes to the HTML of the
+form.
+
+#### Identifying Submit Button
+
+The submit button should be annotated with `test-action="submit"`. This tells
+the test-bot which button should be pressed to continue on in the flow.
+
+Polaris does this automatically for anyone using the Forms stack
+[here](https://github.com/stellar/django-polaris/blob/fd5900d68fec6b0e31ce720262e8d787fcbf8aac/polaris/polaris/templates/withdraw/form.html#L38)
+
+#### Example of fully annotated form
+
+```
+<form action="/submit">
+  <input type="text" id="full-name" test-value="Albert Einstein">
+  <input type="text" id="email" test-value="325235@gmail.com" />
+  <input type="submit" test-action="submit" value="Continue">
+</form>
+```
 
-TBD
+With these steps completed, the validation tooling should be able to exercise
+the entirety of your implementation.
```

Comment:
I would also add that the test suite expects pending deposits to be automatically approved. This will be necessary to run E2E tests for both deposit and withdraw.